tests/filter-effects-1.js

```diff
@@ -8,6 +8,33 @@ export default {
 	status: {
 		stability: 'stable',
 	},
+	values: {
+		properties: ['background-image', 'list-style-image', 'border-image-source', 'cursor', 'mask-image', 'mask-border-source', 'shape-outside', 'content'],
+		// https://caniuse.com/css-filter-function
+		// There was a bug in Safari before 18.4,
+		// https://bugs.webkit.org/show_bug.cgi?id=250303
+		'filter()': {
+			links: {
+				tr: '#FilterCSSImageValue',
+				dev: '#FilterCSSImageValue',
+			},
+			tests: [
+				// filter() = filter( [ <image> | <string> ], <filter-value-list> )
+				'filter(conic-gradient(red), invert(1))',
```

Comment:
Single-stop gradients were only added recently, as far as I know. So I wouldn't include those in the tests for `filter()`.

Generally, I don't think it is necessary to test (so many) different gradient syntaxes here. One basic test per gradient function or even just one for gradients in general should be fine.